build.rs

```diff
@@ -0,0 +1,14 @@
+extern crate lrlex;
+extern crate lrpar;
+
+use lrlex::LexerBuilder;
+use lrpar::CTParserBuilder;
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    let lex_rule_ids_map = CTParserBuilder::<u8>::new()
+        .process_file_in_src("lua5_3/lua5_3.y")?;
+    LexerBuilder::new()
```

Comment:
The compiler doesn't like this because `process_file_in_src` returns a more complicated `Result`, while `main` returns `()` if everything is ok. Should I change the return value of `main` to match the one of `process_file_in_src`?

Reply:
Ah, no, then leave it as it is.